src/js/views/admin/list/programs-all_views.js

```diff
@@ -48,7 +48,7 @@ const LayoutView = View.extend({
       </div>
     </div>
     <div class="flex-region list-page__list">
-      <table><tr>
+      <table class="w-100 js-list-header"><tr>
```

Comment:
does this need the `fixWidth` stuff that the patient list has? Now that it has this header row?

Reply:
is definitely doesn't need the `.js-` hook unless it's used in the js.  I'm surprised it needs the `w-100`  I thought table were 100% by default.. maybe it's a firefox thing?

Reply:
It's 100% a firefox thing
<img width="1552" alt="Screen Shot 2019-10-23 at 2 25 20 PM" src="https://user-images.githubusercontent.com/8376505/67427085-04c3ee80-f5a1-11e9-816d-a9766c05424f.png">

<img width="1552" alt="Screen Shot 2019-10-23 at 2 25 16 PM" src="https://user-images.githubusercontent.com/8376505/67427110-11484700-f5a1-11e9-899a-92c5e4dd696e.png">


that being said, we have the `w-100` on the patient list, so I'll just leave this one too

Reply:
yep so `w-100` 👍  `js-list-header` 👎